packages/nulp_elite/src/pages/content/CategoryPage.js

```diff
@@ -120,7 +120,7 @@ const CategoryPage = () => {
     } finally {
     }
     try {
-      const url = `http://localhost:3000/api/framework/v1/read/nulp?categories=board,gradeLevel,medium,class,subject`;
+      const url = `/api/framework/v1/read/nulp?categories=board,gradeLevel,medium,class,subject`;
```

Comment:
@snehalsabade Framework is hardcoded cc @paritshivani

Reply:
framework - Need to store in zustand store